maestro-cli/src/main/java/maestro/cli/App.kt

```diff
@@ -72,52 +73,50 @@ class App {
     var deviceId: String? = null
 
     companion object {
-        init {
-            AnsiConsole.systemInstall()
-        }
-    }
-}
-
-private fun printVersion() {
-    val props = App::class.java.classLoader.getResourceAsStream("version.properties").use {
-        Properties().apply { load(it) }
-    }
-
-    println(props["version"])
-}
-
-@Suppress("SpreadOperator")
-fun main(args: Array<String>) {
-    val logger = DebugLogStore.loggerFor(App::class.java)
-
-    val commandLine = CommandLine(App())
-        .setUsageHelpWidth(160)
-        .setCaseInsensitiveEnumValuesAllowed(true)
-        .setExecutionExceptionHandler { ex, cmd, _ ->
-            val message = if (ex is CliError) {
-                ex.message
-            } else {
-                ex.stackTraceToString()
+        private fun printVersion() {
+            val props = App::class.java.classLoader.getResourceAsStream("version.properties").use {
+                Properties().apply { load(it) }
             }
 
-            logger.info(message)
-            println()
-            cmd.err.println(
-                cmd.colorScheme.errorText(message)
-            )
-
-            1
+            println(props["version"])
         }
 
-    val exitCode = commandLine
-        .execute(*args)
-
-    DebugLogStore.finalizeRun()
+        @JvmStatic
+        fun main(args: Array<String>) {
```

Comment:
Is moving the main method into the App class necessary?

Reply:
Yeah let's keep as it was before so it's clear what the important changes are in this PR

Reply:
It seems strange to have both an App class and a AppKt class. The AppKt class of course magically inserted by the compiler to capture the free methods.

Reply:
Alright. I'll move it back